### DISCUSSION TITLE
Add Some free usefull tools in Sidebar

### DISCUSSION THREAD

**Discussion by alr86** [end-4/dots-hyprland#892] [Ideas]
Add some free useful APIs to sidebar,
like tools for upload file(e.g. [Send](https://gitlab.com/timvisee/send)), translate(e.g. [Mozhi](https://codeberg.org/aryak/mozhi/)) and share text(e.g. [PrivateBin](https://privatebin.info))

**Comment by jabbawocky** [comment]
**🚀 Launching in a few hours — June 10 midnight PST.**

If you want to support ProposalCraft on Product Hunt tonight, install it now so you have something real to upvote:

```bash
npx @jabbawocky/proposalcraft
```

Then point Claude Desktop at it and try `draft_proposal` with any brief you have sitting around.

The upvote link will be live at midnight — will post it here as soon as it is. Thanks for following along. 🙏
